build.gradle

```diff
@@ -10,6 +11,7 @@ sourceCompatibility = 11
 
 repositories {
     mavenCentral()
+    jcenter()
```

Comment:
Good idea - I'll immediately backport this into master so that this can be in a separate commit.